supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -2,6 +2,14 @@
 [[general-formatting]]
 = General formatting
 
+[[date-formats]]
+== Date formats
+
+Red Hat writers follow _The IBM Style Guide_ for date formats. This list enumerates exceptions:
```

Comment:
* Can just start with "Follow..."
* I would specify what IBM's guidance is here, so "Follow _The IBM Style Guide_  guidance of `day Month year` for date formats."
* Since there is currently only one exception, I would remove this second sentence about the list and make the following paragraph not a bullet.
* I like the real example you gave below, so maybe do there here too? So maybe:

```
Follow _The IBM Style Guide_  guidance of `day Month year` for date formats, for example 3 October 2019.
```